src/screens/DiscoveryScreenV2/elements/DiscoveryItemList.js

```diff
@@ -213,7 +272,10 @@ DomainList.propTypes = {
   DefaultImage: PropTypes.element,
   isCommunity: PropTypes.bool,
   handleSetFollow: PropTypes.func,
-  handleSetUnFollow: PropTypes.func
+  handleSetUnFollow: PropTypes.func,
+  isBlockedSection: PropTypes.bool,
+  handleSetBlock: PropTypes.func,
+  handleSetUnblock: PropTypes.func
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of PropTypes for `isBlockedSection`, `handleSetBlock`, and `handleSetUnblock` is a good practice as it helps in type checking and can prevent potential bugs due to incorrect prop types.

```diff
+ isBlockedSection: PropTypes.bool,
+ handleSetBlock: PropTypes.func,
+ handleSetUnblock: PropTypes.func
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->